update.py

```diff
@@ -166,16 +188,16 @@ def should_rebuild(combo_image, base_images):
 def combine_image_name_and_tag(images):
     name = '_'.join(i.image.split(':')[0] for i in images)
     tag = '_'.join(i.image.split(':')[1] for i in images)
+
     return f'combos/{name}:{tag}'
 
 
 def log_stream(stream):
-    for lines in stream:
-        for line in lines.decode('utf-8').strip().split('\n'):
-            line = json.loads(line)
-            if line.get('errorDetail'):
-                raise DockerBuildError(line['errorDetail'].get('message', str(line)))
```

Comment:
Does catching `python_on_whales.exceptions.DockerException` from `buildx` replace this?

Reply:
It does indeed!